meshcloud-gate-resource.yml

```diff
@@ -0,0 +1,6 @@
+name: gate-resource
```

Comment:
we intend to have the `name` reflect the name of the resource type that would be used in the pipeline. In this case I am assuming it would be just `gate`. If that is true, do you mind changing this to `gate`?

Reply:
No, not a problem. We were just following the perceived naming convention at the time of calling things "*.-resource" when we came up with it. No problem just calling it "gate" because that's what it does (even though it's a rather smart gate ;-) )